apps/emqx_authn/rebar.config

```diff
@@ -34,4 +34,6 @@
 {cover_opts, [verbose]}.
 {cover_export_enabled, true}.
 
+{erl_first_files, ["src/emqx_authentication.erl"]}.
```

Comment:
This module defines a behaviour for providers which are in the same app. We have compile error otherwise.

This will be unnecessary once providers go away from this app.